src/main/java/carpet/script/value/NumericValue.java

```diff
@@ -56,7 +56,7 @@ public String getString()
         }
         try
         {
-            if (Double.isInfinite(value)) return "INFINITY";
+            if (Double.isInfinite(value)) return (value>0)?"+INFINITY":"-INFINITY";
```

Comment:
`+INFINITY` is not a thing. Its `INFINITY` or `-INFINITY`

Can also do `(value > 0 ? "" : "-") + "INFINITY"`